atlp-rwanda/trojans-ec-bn
# #184347082 - Setting up docker

### What does this PR do?

This PR contains configuration files used to set up docker and the documentation needed to have it up and running on your local machine

### Description of Task to be completed?

Using docker compose, set up a ymal file that will run two services
1. express server
2. postgress db.

Note: there is nothing worry on how heroku will run the app. this is only valid when we run locally, heroku will run the application in normal way

### How should this be manually tested?

Check in the readme file how to set it up and run the docker container in your local machine

### Any background context you want to provide?

This is a chore

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2628461/stories/184347082


### Screenshots (if appropriate)


### Questions:

# Type of change

Please check the options that are relevant.


# Checklist:
